tests/functional/fixtures/status/prefix_skel/expected.yaml

```diff
@@ -13,7 +13,8 @@ Prefix:
         eth1: {ip: !!python/unicode '192.168.202.2', network: !!python/unicode 'n1'}
       VNC port: null
       distro: !!python/unicode 'cirros'
-      metadata: {}
+      metadata: !!python/object/apply:collections.OrderedDict
+      - []
```

Comment:
I think it's ok, the array content (the second line) is the contents of the ordered dict, probably we might want to add also a test where there's actually content there to make sure that's actually ordered, though the name of the class there might be enough.